### PR TITLE
Group together shell error strings

### PR DESCRIPTION
**Pull request by FeralChild64**
String-related cleanup:
- grouped all the 'generic' shell strings - this way they will be easier to find when developing command/program
- made their IDs uniform
- dropped one string from MOUSECTL, add newline at the end of its error messages
- fixed one mistake in Polish translation

No action from translators should be necessary.

### CODE REVIEW

**Review by kcgen — APPROVED** [review]
Excellent systematic cleanup, @FeralChild64 !

All looks good from my side.

File: src/dos/program_mousectl.cpp [hunk 1]
```diff
@@ -541,24 +541,25 @@ void MOUSECTL::AddMessages()
...
+	        "No mouse interfaces available.\n");
 	MSG_Add("PROGRAM_MOUSECTL_MISSING_INTERFACES",
-	        "Mouse interface not available.");
-	MSG_Add("PROGRAM_MOUSECTL_NO_PHYSICAL_MICE", "No physical mice detected.");
+	        "Mouse interface not available.\n");
+	MSG_Add("PROGRAM_MOUSECTL_NO_PHYSICAL_MICE",
+	        "No physical mice detected.\n");
 	MSG_Add("PROGRAM_MOUSECTL_NO_MATCH",
-	        "No available mouse matching the pattern found.");
+	        "No available mouse matching the pattern found.\n");
```
Comment (johnnovak) [inline src/dos/program_mousectl.cpp:562]: No available mouse found matching the pattern.

File: src/shell/shell.cpp [hunk 2]
```diff
@@ -944,8 +944,18 @@ static char const * const full_name="Z:\\COMMAND.COM";
 static char const * const init_line="/INIT AUTOEXEC.BAT";
 
 void SHELL_Init() {
-	/* Add messages */
-	MSG_Add("SHELL_ILLEGAL_PATH","Illegal Path.\n");
+	// Generic messages, to be used by any command or DOS program XXX
+	MSG_Add("SHELL_ILLEGAL_PATH", "Illegal path.\n");
+	MSG_Add("SHELL_ILLEGAL_SWITCH", "Illegal switch: %s.\n");
+	MSG_Add("SHELL_MISSING_PARAMETER", "Required parameter missing.\n");
+	MSG_Add("SHELL_SYNTAX_ERROR", "The syntax of the command is incorrect.\n");
```
Comment (johnnovak) [inline src/shell/shell.cpp:951]: Incorrect command syntax

**Review by johnnovak — APPROVED** [review]
Nice cleanup job @FeralChild64 👍🏻

File: src/dos/program_mousectl.cpp (hunk 1 above)
Comment (FeralChild64) [reply src/dos/program_mousectl.cpp]: OK, changed.

File: src/shell/shell.cpp (hunk 2 above)
Comment (FeralChild64) [reply src/shell/shell.cpp]: OK, changed.

File: src/shell/shell.cpp [hunk 3]
```diff
@@ -944,8 +944,18 @@ static char const * const full_name="Z:\\COMMAND.COM";
...
+	// Generic messages, to be used by any command or DOS program
+	MSG_Add("SHELL_ILLEGAL_PATH", "Illegal path.\n");
+	MSG_Add("SHELL_ILLEGAL_SWITCH", "Illegal switch: %s.\n");
+	MSG_Add("SHELL_MISSING_PARAMETER", "Required parameter missing.\n");
+	MSG_Add("SHELL_SYNTAX_ERROR", "Incorrect command syntax.\n");
+	MSG_Add("SHELL_ACCESS_DENIED", "Access denied - %s\n");
+	MSG_Add("SHELL_FILE_CREATE_ERROR", "File creation error - %s\n");
+	MSG_Add("SHELL_FILE_OPEN_ERROR", "File open error - %s\n");
+	MSG_Add("SHELL_FILE_NOT_FOUND", "File not found: %s\n");
+	MSG_Add("SHELL_FILE_EXISTS", "File %s already exists.\n");
```
Comment (johnnovak) [inline src/shell/shell.cpp:956]: Consider enclosing filenames and strings with single quotes in the messages — I think it's nicer.

Comment (FeralChild64) [reply src/shell/shell.cpp]: OK, changed.

**Comment by kcgen** [comment]
Another big clean up -- nice, @FeralChild64 !

@MeAreJeenius , this touches some of Shell and its translation messages (but not Shell Batch); fingers crossed your branch rebases clean once this lands in `main`.

**Pull request merged**